solutions/project/hub-env/fortigate/address.yaml

```diff
@@ -0,0 +1,196 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# IP reservation
+## Fortigate External VPC IP reservation
+## Primary
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:
+  name: hub-fgt-primary-ext-address
+  namespace: networking
+  annotations:
+    cnrm.cloud.google.com/project-id: hub-project-id # kpt-set: ${hub-project-id}
+    config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/networking/ComputeSubnetwork/hub-nane1-external-paz-snet
+spec:
+  resourceID: fgt-primary-ext-address
+  address: 172.31.200.10
```

Comment:
Does this need to be modified or is it static across all deployments?

Reply:
this is a private ip so it can remain static across deployments